GSS-Cogs/idpd-api-poc
# Data structures

Updates the stub api and all dependant logic and tests to use the agreed data structures.

## How to review

- Pull this branch
- `pipenv sync --dev`
- `export LOCAL_BROWSE_API=true` - this will allow browser traffic despite mimetypes and will replace all `staging.idpd.uk` values with `localhost:8000` to allow click through navigation.
- `make start`
- go to http://localhost:8000/datasets

have a play around, make sure it looks sane and that things resolve to sensible json representations.

**Note** - there's a few TODO's and commented out code sections here relating to the graph, these will be covered in following tasks, this is already too big (and the graphs not relevant to the "make it representative" goal here).
